src/DevChatter.Bot.Core/Systems/Streaming/FollowableSystem.cs

```diff
@@ -27,8 +31,9 @@ private void FollowerServiceOnOnNewFollower(object sender, NewFollowersEventArgs
             {
                 foreach (string followerName in eventArgs.FollowerNames)
                 {
+                    _currencyGenerator.AddCurrencyTo(followerName, TOKENS_FOR_FOLLOWING);
                     chatClient.SendMessage(
-                        $"Welcome, {followerName}! Thank you for following! Everyone, say \"hello\"!");
+                        $"Welcome, {followerName}! Thank you for following! Here's  {TOKENS_FOR_FOLLOWING} coins to have some fun. Everyone, say \"hello\"!");
```

Comment:
Oops, well that's a quick fix. Not used to git, can I change things and add them to the existing pull request?

Reply:
Yep if you push a commit to the master branch in your fork (as that's the branch you used for this pull request) it will just automatically show up under this PR